plugins/reporters/trustsource/src/main/kotlin/TrustSourceReporter.kt

```diff
@@ -69,27 +67,34 @@ private fun traverseDeps(input: ReporterInput, deps: Sequence<DependencyNode>):
             input.ortResult.getPackageLicenseChoices(dep.id),
             input.ortResult.getRepositoryLicenseChoices()
         )
+
         val licenses = effectiveLicense?.decompose()?.map {
             val name = it.toString()
             val url = it.getLicenseUrl().orEmpty()
 
-            TrustSourceLicense(name, url)
+            License(name, url = url)
```

Comment:
Nit: There is no gain in using a named `url =` argument here, and it should be simply omitted.

Reply:
Done